nucypher/characters/control/interfaces.py

```diff
@@ -138,7 +138,7 @@ def revoke(self, label: bytes, bob_verifying_key: bytes) -> dict:
         if len(failed_revocations) > 0:
             for node_id, attempt in failed_revocations.items():
                 revocation, fail_reason = attempt
-                if fail_reason == NotFound:
+                if fail_reason == NotFound:  # FIXME
```

Comment:
What does need a fix here? Should we open an issue or add a TODO?

Reply:
:heavy_check_mark: